frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -64,13 +64,14 @@ export function Experiments(): JSX.Element {
         },
         createdByColumn<Experiment>() as LemonTableColumn<Experiment, keyof Experiment | undefined>,
         createdAtColumn<Experiment>() as LemonTableColumn<Experiment, keyof Experiment | undefined>,
+        atColumn('start_date', 'Started') as LemonTableColumn<Experiment, keyof Experiment | undefined>,
         {
             title: 'Duration',
             key: 'duration',
             render: function Render(_, experiment: Experiment) {
                 const duration = getExperimentDuration(experiment)
 
-                return <div>{duration !== undefined ? `${duration} day${duration !== 1 ? 's' : ''}` : '--'}</div>
+                return <div>{duration !== undefined ? `${duration} day${duration !== 1 ? 's' : ''}` : '—'}</div>
```

Comment:
why change this?

Reply:
`atColumn` uses an em dash for the empty state, so wanted to ensure this looks consistent :)